framework/src/org/checkerframework/common/value/qual/MinLenFieldInvariant.java

```diff
@@ -1,4 +1,4 @@
-package org.checkerframework.checker.index.qual;
+package org.checkerframework.common.value.qual;
```

Comment:
I'm not sure how to change this, since I don't really understand what it's doing in the first place. Should I change the name, but leave it so that it only accepts a lower bound (i.e. a MinLen)? That doesn't seem like the right thing to do, but I'm not sure what I should add to make it able to handle all of ArrayLenRange.